### PR TITLE
Port extension to Microsoft Edge

### LINKED ISSUE #14: Look into porting this to Microsoft Edge
I will look into this when I have time (probably in about 1-2 weeks).

### PR DESCRIPTION
**Pull request by rik-smeets**
Resolves #14.

The port includes: 
- Files generated by generated by the Microsoft Edge Extension Toolkit, allowing Chrome extensions to work in Edge. This includes a slightly extended `manifest.json`. 
- Some extra icons necessary in these dimensions as store icons for the Microsoft Store. 
- Updated styling: Edge did not handle the current styling correctly, so I changed the way the icon is rendered. It works fine in other browsers as well.

I also tested the extension in Chrome and Firefox after these changes, and it still works as expected in those browsers too. 

Good news: Microsoft approved my request to publish this extension to the Microsoft Store, which is currently restricted. I just submitted the extension for certification; afterwards, I will manually choose when it can actually be published to the Store (in case there are some review comments here which would block its release).

### CODE REVIEW

File: src/search_results.css [hunk 1]
```diff
@@ -3,7 +3,7 @@
 }
 
 .highlighted-search-result::before {
-  content: "►";
+  content: "\25BA";
```
[the remark this answers is not in the document]
Comment (rik-smeets) [reply src/search_results.css]: Correct, is is not supported. This does work and also still works in the other supported browsers.

**Comment by rik-smeets** [comment]
@infokiller Thanks for your feedback, I'll work on it as soon as possible. Might be a couple of days since I'm busy this weekend.

File: src/manifest.json [hunk 2]
```diff
@@ -1,218 +1,233 @@
 {
```
[the remark this answers is not in the document]
Comment (rik-smeets) [reply src/manifest.json]: Done. The difference is now much clearer.

**Comment by infokiller** [comment]
Thanks @rik-smeets!
I will probably reformat the manifest a bit (I like to use an auto formatter because I hate doing formatting manually), but I can just do it in a followup commit.

**Pull request merged**